src/bytecode/concrete.py

```diff
@@ -969,14 +979,21 @@ def to_bytecode(
         # Free vars are never shared and correspond to index larger than the
         # largest cell var.
         # See PyCode_NewWithPosOnlyArgs
-        if sys.version_info >= (3, 11):
+        if PY311:
             cells_lookup = self.varnames + [
-                n for n in self.cellvars if n not in self.varnames
+                CellVar(n) for n in self.cellvars if n not in self.varnames
             ]
             ncells = len(cells_lookup)
         else:
             ncells = len(self.cellvars)
-            cells_lookup = self.cellvars
+            cells_lookup = [CellVar(n) for n in self.cellvars]
+
+        # In Python 3.13+ LOAD_FAST can be used to retrieve cell values
+        locals_lookup: Sequence[Union[str, CellVar]]
+        if PY313:
+            locals_lookup = cells_lookup
```

Comment:
```suggestion
            locals_lookup = (
                self.varnames
                + [FreeVar(n) for n in self.freevars if n not in self.varnames]
                + [CellVar(n) for n in self.cellvars if n not in self.varnames]
            )
```

Reply:
Why does the free vars appear before the cell vars ?

Reply:
With this change I can repro the 3.13 framework failures on this branch

Reply:
> Why does the free vars appear before the cell vars ?

All I have is empirical evidence at the moment

Reply:
From this the cellvars should be packed before the freevars though, no? 🤔 
https://github.com/P403n1x87/cpython/blob/93156880efd14ad7adc7d3512552b434f5543890/Objects/codeobject.c#L728-L760

Reply:
Ah! Swapping the order of freevars and cellvars seems to work too! 😮

Reply:
You can reproduce or you can fix the issue ?